sycl/doc/extensions/experimental/sycl_ext_oneapi_annotated_arg.asciidoc

```diff
@@ -210,6 +210,9 @@ class annotated_arg {
     operator T() noexcept;
     operator T() const noexcept;
 
+    // Available if type T is pointer type
+    T operator->() const noexcept;
```

Comment:
This doesn't match the implementation. There it is `T*` return type. Which one is intended?

Reply:
This `T` in the docs should match the whole type, while in the implementation there are two subclass. That `T` is after removing the *